pkgs/applications/editors/emacs/make-emacs.nix

```diff
@@ -213,6 +215,8 @@ mkDerivation (finalAttrs: {
   ] ++ lib.optionals srcRepo [
     autoreconfHook
     texinfo
+  ] ++ lib.optionals withCompressInstall [
+    gzip
```

Comment:
I believe gzip is in the stdenv, this is unnecessary.

Reply:
[You're right](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md?plain=1#L172). I'll remove that then; thanks for giving me the words to look this up